docs/ingest-and-test.md

```diff
@@ -0,0 +1,87 @@
+# Testing local installation
+
+These instructions will lead you through some basic functionality tests, ingesting some sample data, and running some tests of the data services to make sure your local installation is working as expected.
+
+## Initial tests
+
+Check that you can see the data portal in your browser at `http://docker.localhost:5080/`.
+
+Check that you can generate a bearer token for user2 with the following call, substituting secrets and passwords from `tmp/secrets/keycloak-client-local_candig-secret` and `tmp/secrets/keycloak-test-user2-password`.
+
+```
+## user2 bearer token
+curl -X "POST" "http://docker.localhost:8080/auth/realms/candig/protocol/openid-connect/token" \
+     -H 'Content-Type: application/x-www-form-urlencoded; charset=utf-8' \
+     --data-urlencode "client_id=local_candig" \
+     --data-urlencode "client_secret=<client_secret" \
+     --data-urlencode "grant_type=password" \
+     --data-urlencode "username=user2" \
+     --data-urlencode "password=<user2-password>" \
+     --data-urlencode "scope=openid"
+```
+
+Doing much else will require test data.
+
+## Install test data
+
+Clone the [candig-ingest](https://github.com/CanDIG/candigv2-ingest) repo:
+
+```
+https://github.com/CanDIG/candigv2-ingest.git
+```
+
+Create a virtual environment named `.venv`:
+
+```
+# Linux
+sudo apt-get install python3-venv    # If needed
+python3 -m venv .venv
+source .venv/bin/activate
+
+# macOS
+python3 -m venv .venv
+source .venv/bin/activate
+
+# Windows
+py -3 -m venv .venv
+.venv\scripts\activate
+```
+
+Install the requirements:
+
+```
+pip install -r requirements.txt
+```
+
+Generate a file env.sh
+
+```
+python settings.py ../CanDIGv2/
+source env.sh
+```
+
+Create opa dataset policy (for example, dataset_name=SYNTHETIC_1 and user_name=user2)
```

Comment:
user name should be updated to the email address

Reply:
Is the deployment / ingest set up for our default users (user1, user2) to have email addresses?

Reply:
done